### ISSUE TITLE
Using only the tab key the user should be able to reach all interactive elements

### ISSUE THREAD

**Issue by blnkt** [ManifoldScholar/manifold#1076]
This includes form elements, links, buttons, and any other elements behaving as such.  This work does not include those sets of components updated in dedicated commits (see related issues below).

Related to #1052, #1111, #1112, #1117, #1119
